README.md

```diff
@@ -17,9 +17,14 @@ private GitLab instance.
 Install it as you would install [fugitive.vim][]
 (you will also need [fugitive.vim][] installed)
 
-To use private GitLab repositories add the following to your .vimrc
+To use private GitLab instances, add the following to your .vimrc
 
-    let g:fugitive_gitlab_domains = ['https://my.gitlab.com']
+    let g:fugitive_gitlab_domains = {'https://my.gitlab.com'}
```

Comment:
Is this correct? That's a dictionary without a key/value.
My vim complains about this syntax.

Reply:
I suspect your middle commit could/should be dropped
1st and 3rd commits look good.

Reply:
if ssh and web urls are the same, the options are:
```
['https://my.gitlab.com'']
" or
{'my.gitlab.com': ''https://my.gitlab.com''}
```
Probably simpler to just leave the README detailing the first option

Reply:
> Is this correct? That's a dictionary without a key/value.
> My vim complains about this syntax.

It's not correct. I've dropped the commit.

> Probably simpler to just leave the README detailing the first option

I agree.